components/Document.js

```diff
@@ -7,13 +7,49 @@ function EditPage({ source: initialDocument }) {
   const [doc, setDoc] = React.useState(initialDocument);
   const { content, config, errors } = useMarkdocCode(doc);
 
+  const [docChanged, setDocChanged] = React.useState(false);
+
+  const [prNumber, setPrNumber] = React.useState(null);
+
+  const createPR = async () => {
+    const filename = window.CURRENT_FILENAME;
+
+    const files = {
+      [filename]: doc
+    };
+
+    console.log(files);
+
+    const body = JSON.stringify({ files });
+
+    const response = await fetch('/api/create-pr', {
```

Comment:
I feel like we should just call this:
```suggestion
    const response = await fetch('/api/pr', {
```

and `POST` to it.